dev-support/vagrant/README.md

```diff
@@ -13,7 +13,7 @@
 -->
 # What is this?
 
-This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache (Incubating) Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
+This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
```

Comment:
There are quite many other changes required in vagrant.  Let's change them all together in a separated JIRA?
```
-- ./dev-support/vagrant/README.md --------------------------------------------
 16: This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache (Incubating) Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
   1 occurrence(s)
-- ./dev-support/vagrant/Vagrantfile ------------------------------------------
 24: RATIS_PATH = ::File.join(VAGRANT_HOME, 'incubator-ratis')
   1 occurrence(s)
-- ./dev-support/vagrant/bin/start_ratis_load_gen.sh --------------------------
 22: cd ${HOME}/incubator-ratis
   1 occurrence(s)
-- ./dev-support/vagrant/bin/start_ratis_server.sh ----------------------------
 25: cd ${HOME}/incubator-ratis/
   1 occurrence(s)
-- ./dev-support/vagrant/screenrcs/namazu_hdd_screenrc ------------------------
 23: screen -t Disk0 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data0 -mount-point /home/vagrant/test_data/data0_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
 24: screen -t Disk1 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data1 -mount-point /home/vagrant/test_data/data1_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
 25: screen -t Disk2 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data2 -mount-point /home/vagrant/test_data/data2_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
   3 occurrence(s)
-- ./dev-support/vagrant/screenrcs/ratis_ratis-hdd-slowdown_screenrc ----------
 23: screen -t Server0 0 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data0_slowed n0 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 24: screen -t Server1 1 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data1_slowed n1 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 25: screen -t Server2 2 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data2_slowed n2 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 26: screen -t LoadGen 3 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_load_gen.sh n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
   4 occurrence(s)
-- ./dev-support/vagrant/screenrcs/ratis_ratis-server_screenrc ----------------
 23: screen -t Server0 0 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data0 n0 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 24: screen -t Server1 1 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data1 n1 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 25: screen -t Server2 2 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data2 n2 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 26: screen -t LoadGen 3 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_load_gen.sh n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
   4 occurrence(s)
```

Reply:
Sure